### ISSUE TITLE
Add documentation and handlebars validation for helpers

### ISSUE THREAD

**Issue by andrewstucki** [elastic/package-registry#603]
We just added the first handlebars helper for use in packages in https://github.com/elastic/kibana/pull/72698 in order to support some beats filebeat module conversion to packages. We should have:

1. documentation for the available helpers here for anyone writing a new package, and
2. code for validating the handlebars templates prior to being pushed down to Kibana.

**Comment by ruflin** [comment]
@ycombinator I would consider something like above to be part of the package spec. WDYT?

**Comment by ycombinator** [comment]
Agreed. Let's move this issue to the package spec repo so we can make sure this work is done there.
